src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -749,7 +824,7 @@ async Task GetStreamIdWithoutStartWorks()
             {
                 (QuicConnection clientConnection, QuicConnection serverConnection) = await CreateConnectedQuicConnection();
 
-                using QuicStream clientStream = clientConnection.OpenBidirectionalStream();
+                using QuicStream clientStream = await clientConnection.OpenBidirectionalStreamAsync();
```

Comment:
Ha, I guess `GetStreamIdWithoutStart` doesn't fully make sense now? As there's no way to create a stream without notifying the peer (we now add `PEER_ACCEPT` flag; and earlier we would only flush stream creation to peer with the first write I think? If so, we can simplify some tests that make sure stream is flushed to peer by performing a write, see e.g. https://github.com/dotnet/runtime/blob/f6ca720b4276e30b1494400dd11b3e704286aba3/src/libraries/System.Net.Quic/tests/FunctionalTests/QuicTestBase.cs#L263-L265 )

This comment is *not* for your action @rzikm, it is just a thing I noticed and wanted to point out so we can think this through later

cc @ManickaP

Reply:
I don't think we have changed whether peer gets notified or not. the `INDICATE_PEER_ACCEPT` flag does not indicate sending any signal to peer, just that we want to be notified once we are within advertised stream limits. The immediate notification of the peer is set by `QUIC_STREAM_START_FLAG_IMMEDIATE` flag.

Reply:
Ok, the naming is confusing 😄